app/Http/routes.js

```diff
@@ -15,6 +15,23 @@
 | Route.resource('user', 'UserController')
 */
 
-const Route = use('Route')
+const Route = use('Route');
+const fetch = require('node-fetch');
 
-Route.on('/').render('welcome')
+
+Route.get('/', function * (request, response) {
+  yield response.sendView('form');
```

Comment:
Since you are sending the view right here the data from the api isn't put into the view.

Instead move this where you have `response.send` and then use `response.sendView('form', {formInputs: data})` to get the `formInputs` array into your template.